Formula/bash-preexec.rb

```diff
@@ -1,5 +1,5 @@
 class BashPreexec < Formula
-  desc "preexec and precmd functions for Bash just like Zsh."
```

Comment:
Again, I reckon a whitelist of this name may make sense. I'd put the `(just like Zsh)` because the reordering changes the meaning a bit.